src/flattenArray.ts

```diff
@@ -0,0 +1,5 @@
+export default function flattenArray(list: any[]): any[] {
```

Comment:
only used in one place, does not need to be in it's own file

Reply:
Removed the file anyway 👍